server/lib/systemd/pbench-server.service

```diff
@@ -10,12 +10,11 @@ After=network.target httpd.service
 
 [Service]
 Type = simple
+User = pbench
```

Comment:
I wonder if you should specify `Group = pbench` as well? Aside from that, this looks good!

Reply:
yeah we can add `Group = pbench` as well, I was just not sure if user pbench should be part of group pbench or some other group. Usually user by default is in the same group as its name so adding a `Group=` is fine.

Reply:
fixed